CMakeLists.txt

```diff
@@ -1637,7 +1637,10 @@ if (CMAKE_CROSSCOMPILING)
   set(ZLIB_FOUND OFF)
 else ()
   find_package(ZLIB)
-  find_library(libsnappy snappy)
+  # On Ubuntu 14.10, 32-bit builds fail to link with -lsnappy, just ignore.
```

Comment:
Is there just a separate package to install for 32-bit libsnappy?

Reply:
Hmm, looks like in 14.04 the package isn't properly multiarch (it always tries to go to /usr/lib [1][2]), so the 32-bit and 64-bit versions can't coexist.

Newer versions are fine [3] [4], but doesn't seem worth the trouble to deal with.

[1] https://packages.ubuntu.com/trusty/amd64/libsnappy1/filelist
[2] https://packages.ubuntu.com/trusty/i386/libsnappy1/filelist
[3] https://packages.ubuntu.com/cosmic/amd64/libsnappy1v5/filelist
[4] https://packages.ubuntu.com/cosmic/i386/libsnappy1v5/filelist